### ISSUE TITLE
`Make build` with problem

### ISSUE THREAD

**Issue by gabrielbdornas** [transparencia-mg/dtamg-docker#7]
```
Iniciando ambiente...
root@40a3b5929a75:/dataset# make build
Construindo datapackage.json...
Traceback (most recent call last):
  File "/usr/local/bin/frictionless", line 5, in <module>
    from frictionless.__main__ import program
  File "/usr/local/lib/python3.9/site-packages/frictionless/__init__.py", line 20, in <module>
    from .program import program
  File "/usr/local/lib/python3.9/site-packages/frictionless/program/__init__.py", line 1, in <module>
    from .api import program_api
  File "/usr/local/lib/python3.9/site-packages/frictionless/program/api.py", line 1, in <module>
    from typer import Option as Opt
  File "/usr/local/lib/python3.9/site-packages/typer/__init__.py", line 12, in <module>
    from click.termui import get_terminal_size as get_terminal_size
ImportError: cannot import name 'get_terminal_size' from 'click.termui' (/usr/local/lib/python3.9/site-packages/click/termui.py)
make: *** [Makefile:42: datapackage.json] Error 1
```

**Comment by gabrielbdornas** [comment]
The solution suggested [here](https://github.com/tiangolo/typer/issues/377#issuecomment-1081506095) is to downgrad click packate.

**Comment by tiangolo** [comment]
I just released Typer `0.4.1` that should handle it. :rocket: :nerd_face:
